index.bs

```diff
@@ -4902,6 +5002,56 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the [=public key credential source=] created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
```

Comment:
I hadn't noticed that this was also defined at authentication time. What use did you have in mind for that?

Reply:
The initial idea was that it could allow possible discovery of the type of credential for existing registrations created under L1 if the client is able to determine that. I guess this might be possible for platform authenticators, but probably not for portable authenticators. If other credential properties (i.e. new properties beyond `rk`) are supported by a client, then it might have other uses.

If you see no value in it, I've no issues with dropping the authentication usage.

Reply:
I would drop it for now because a) I don't know what I would do with it and b) it can always be added later if we add a property where it makes clear sense.

Reply:
Fair enough - removed.

Reply:
Dropped use of this extension for assertion flow

Reply:
For what it's worth, leaving it in would mean authenticators can start implementing it in authenticatorGetAssertion. It doesn't have to be left to the client to figure out what to do with it.